snapcraft/internal/lxd.py

```diff
@@ -50,7 +51,8 @@ def _pull_file(self, src, dst):
                     '{}/{}'.format(self._container_name, src), dst])
 
     def _container_run(self, cmd):
-        check_call(['lxc', 'exec', self._container_name, '--'] + cmd)
+        check_call(['lxc', 'exec', self._container_name, '--'] + cmd,
+                   env=self._env)
```

Comment:
This sets the environment on the _host_ for this command, doesn't it? Does that get propagated into the `lxc exec`, or should this be set within the command itself?